scripts/joinmarket-qt.py

```diff
@@ -2026,6 +2033,15 @@ def updateWalletInfo(self):
         elif not self.wallet_service.synced:
             return
         else:
+            # sync phase is finished; we now wait for callbacks:
+            if self.walletRefresh and self.walletRefresh.running:
+                self.walletRefresh.stop()
+                self.walletRefresh = None
```

Comment:
I feel the usage of `walletRefresh` makes the code more complicated than necessary, specifically, the fact that the logic of starting & stopping it is in two different places makes it hard to follow.

Reply:
See above. Not elegant but it or something similar is needed.